src/data/sql-quiz.ts

```diff
@@ -831,20 +831,22 @@ const sqlQuiz = [
     Link: "https://www.freecodecamp.org/news/a-beginners-guide-to-sql/"
   },
   {
-    Question:
-      "What SQL statement contains a valid subquery?",
-    Answer: "SELECT * FROM employees WHERE wage > (SELECT AVG(wage) FROM employees)",
+    Question: "What SQL statement contains a valid subquery?",
```

Comment:
Please remove all of these changes in this file because you already have them in your other PR.
So we don't need it twice here 👍